build.sh

```diff
@@ -18,4 +18,7 @@ em++ \
 	-s NO_EXIT_RUNTIME=1 \
 	-s EXPORTED_FUNCTIONS="['_Hunspell_create', '_Hunspell_destroy', '_Hunspell_spell', '_Hunspell_suggest', '_Hunspell_free_list']" \
 	./hunspell/src/hunspell/.libs/libhunspell-1.6.a \
-	-o embuild/hunspell.js
\ No newline at end of file
+	-o embuild/hunspell.js
+
+cat LICENSE embuild/hunspell.js > tmp; mv tmp embuild/hunspell.js
+cat LICENSE embuild/hunspell.js.mem > tmp; mv tmp embuild/hunspell.js.mem
```

Comment:
Instead of creating a new directory I would make a `dist/` folder and copy directly there like `cat LICENSE embuild/hunspell.js > dist/hunspell.js`